### DISCUSSION TITLE
Welcome to easylist Discussions!

### DISCUSSION THREAD

**Discussion by ryanbr** [easylist/easylist#13025] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by wavygecko** [comment]
Hello! I thought I'd start this thread off by giving a big thank you to @ryanbr and all the other contributors here. The time you all put into this project to make the web a little nicer is appreciated by so many. 😊
